src/style-installer.js

```diff
@@ -159,12 +159,14 @@ export function makeBodyVisible(doc, opt_waitForExtensions) {
       }
     }
   };
-  waitForBody(doc, () => {
-    const perf = performanceFor(doc.defaultView);
+  const win = doc.defaultView;
+  const docState = documentStateFor(win);
+  docState.onBodyAvailable(() => {
```

Comment:
Oh. Actually this is fine since, it looks like, we try to create `DocumentState` each time.